themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_de.properties

```diff
@@ -85,11 +85,13 @@ notSetUp={0} ist nicht eingerichtet.
 two-factor=Zwei-Faktor Authentifizierung
 passwordless=Kennwortlos
 unknown=Unbekannt
+resetTotpMissingIdError=Bitte w�hlen Sie eine OTP Konfiguration aus.
```

Comment:
Looks like the Umlaute are usually escaped in these files. So this needs to be `\u00E4`.

Reply:
You're right. Fixed !